### PR TITLE
Ngo detail page better UX

### LINKED ISSUE #110: Page jump on Needs next page buttons
When I use these buttons my page jumps back to top. it should preserve its scroll position and remain locked at that frame

### PR DESCRIPTION
**Pull request by costibleotu**
### What does it fix?
NGO detail:
*  Hiding forms when success messages
* Opens current kind accordion
* Scrolls to active element ( success message, form or switched page kind)
* Opens donations and report accordions 

NAV:
* goto admin after login

Closes #110 #108

### CODE REVIEW

File: ro_help/hub/templates/ngo/donate.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {% block kind-kind-filters %}
 {% endblock%}
 
-<div class="container">
+<div class="container">xxxx
```
Comment (catileptic) [inline ro_help/hub/templates/ngo/donate.html:11]: Is this a typo?

File: ro_help/ro_help/settings/dev.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 AUTH_PASSWORD_VALIDATORS = []
 
-# EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
```
Comment (catileptic) [inline ro_help/ro_help/settings/dev.py:11]: This might need to be commented out again.

File: ro_help/hub/templates/ngo/donate.html (hunk 1 above)
Comment (costibleotu) [reply ro_help/hub/templates/ngo/donate.html]: 10x 4 seeing this :). fixed

File: ro_help/ro_help/settings/dev.py (hunk 2 above)
Comment (costibleotu) [reply ro_help/ro_help/settings/dev.py:11]: we should fix it the right way :)

**Comment by costibleotu** [comment]
@catileptic fixed the redirect after login. pls try now

**Pull request merged**